src/main/java/tsundere/task/Task.java

```diff
@@ -0,0 +1,61 @@
+package tsundere.task;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * Initializes Task with its name.
+     * Completion status is false by default.
+     *
+     * @param description Name of Task.
+     */
+    public Task(String description) {
+        this.description = description;
```

Comment:
Consider running the `trim()` command so that you remove the trailing and leading whitespace.